### ISSUE TITLE
Regular Update for Dependencies

### ISSUE THREAD

**Issue by mjones-oddball** [department-of-veterans-affairs/notification-api#1339]
## User Story - Business Need

We wish to keep dependencies up to date so we do not need such massive overhauls of our system. This will be a recurring ticket that will be done every sprint. We will update all dependencies we are able to. Any conflicts will get a ticket. This is intended to be a day of work at most because this is intended to update with only non-breaking changes.

If possible, see if we can upgrade mistune to > 2.0.2, which will clear team-709.

- [x] Ticket is understood and QA has been contacted

### User Story
**As** VA Notify
**I want** to keep our service up to date
**So that** we are secure and as free of bugs as possible

### Additional Info and Resources
- Relevant section of [README.md](https://github.com/department-of-veterans-affairs/notification-api#update-requirementstxt)
- [ticket to upgrade mistunes](https://app.zenhub.com/workspaces/va-notify-620d21369d810a00146ed9c8/issues/gh/department-of-veterans-affairs/notification-utils/129)

## Engineering Checklist
- [x] Update performed
- [x] Any conflicts removed and tickets created
- [x] Passes all tests locally
- [x] Passes userflows and QA Suite testing against Dev
- [x] Any major changes are investigated (compare existing to new version changes)
  - [x] Any questionable version changes are removed and a ticket is created
- [x] Tickets created are added to the Epic for tracking purposes

## Acceptance Criteria
Repo dependencies are updated and we have no broken functionality. Tickets created for any updates we could, or should, not do.

## QA Considerations
- [x] Check to see if these updates cancel out any Twistlock issues
- [x] QA Regression tests pass after deploying this code.

**Comment by kalbfled** [comment]
@mjones-oddball Upgrading mistune should be a notification-utils ticket because it is specified [here](https://github.com/department-of-veterans-affairs/notification-utils/blob/master/setup.py#L35).  Once that upgrade is complete and merged into utils master, the next routine API dependency upgrades will pull in the newer version of mistune.

**Comment by cris-oddball** [comment]
mistune is specified in `notification-api/requirements.txt` - no idea what the utils repo is doing with it.

**Comment by kalbfled** [comment]
notification-api/requirements.txt includes mistune because it is a subdependency of notification-utils, which is a top-level dependency for API.

notification-api does not import mistune:
```
...notification-api$ grep -rni mistune .
./requirements.txt:65:mistune==0.8.4
```

notification-utils imports mistune:
```
...notification-utils$ grep -rni mistune notifications_utils/
...
notifications_utils/formatters.py:5:import mistune
...
```

**Comment by cris-oddball** [comment]
merged to master and deployed to perf on a cut tag.

**Comment by cris-oddball** [comment]
Perf regression (local) 122 passed, 3 xfailed in 143.10s (0:02:23)

**Comment by cris-oddball** [comment]
@justaskdavidb2 please provide the link to the new ticket you created for UTILs for the mistune upgrade and provide the link.

Not closing ticket until link is provided.

**Comment by cris-oddball** [comment]
Ticket created from this ticket to upgrade mistunes in Utils
[Utils-129](https://app.zenhub.com/workspaces/va-notify-620d21369d810a00146ed9c8/issues/gh/department-of-veterans-affairs/notification-utils/129)

**Comment by cris-oddball** [comment]
awaiting one more ticket for upgrades in setup.py in the utils repo.

**Comment by cris-oddball** [comment]
All good and closing ticket.
